### PR TITLE
Update documentation to reflect emulator is default for integration tests.

### PR DESCRIPTION
**Pull request by thebrianchen**

### CODE REVIEW

File: firebase-firestore/README.md [hunk 1]
```diff
@@ -41,7 +41,34 @@ the source root.
...
 
-### Run on Local Emulator
+By default, integration tests run against the Firestore emulator.
+
+### Setting up the Firestore Emulator
+
+The integration tests require that the Firestore emulator is running on port
+8080, which is default when running it via CLI.
+
+  * Ensure that `gogole-services.json` is properly configured.
```
Comment (wilhuff) [inline firebase-firestore/README.md]: What does this mean? Also typo.

Comment (thebrianchen) [reply firebase-firestore/README.md]: I wish to communicate that the Android SDK is hard-coded to look for the port on 8080, and that port 8080 is the default port that the android emulator will start on when running it via CLI.

Comment (wilhuff) [reply firebase-firestore/README.md]: I should have been more specific with my feedback: what does it mean to "ensure that `google-services.json` is properly configured"? Do I need to do something special? If this is the standard setup maybe just link to https://github.com/firebase/firebase-android-sdk#project-setup ?

Also, if we're running against the emulator, why do we need the `google-services.json` at all?

Comment (thebrianchen) [reply firebase-firestore/README.md]: Gotcha. I did not realize that the `google-services.json` isn't needed when running against the emulator.

**Pull request merged**